common/beerocks/bwl/CMakeLists.txt

```diff
@@ -79,7 +81,8 @@ elseif(BWL_TYPE STREQUAL "WAV")
     )
 
 elseif(BWL_TYPE STREQUAL "DWPAL")
-
+    
+    set(BWL_TYPE "DWPAL" CACHE STRING "" FORCE)
```

Comment:
Setting BWL_TYPE twice, first as normal variable and then as cache variable, is just confusing.

I think it's better to set local variable BWL_TYPE_DEFAULT in the platform condition, and after the condition add one and only one
```
set(BWL_TYPE ${BWL_TYPE_DEFAULT} CACHE STRING "Which BWL backend to use")
```

Reply:
done